diff --git a/node_modules/handlebars/lib/index.js b/node_modules/handlebars/lib/index.js
index 3071b22..66d8ec6 100644
--- a/node_modules/handlebars/lib/index.js
+++ b/node_modules/handlebars/lib/index.js
@@ -20,7 +20,7 @@ function extension(module, filename) {
   module.exports = handlebars.compile(templateString);
 }
 /* istanbul ignore else */
-if (typeof require !== 'undefined' && require.extensions) {
+if (false && typeof require !== 'undefined' && require.extensions) {
   require.extensions['.handlebars'] = extension;
   require.extensions['.hbs'] = extension;
 }
